src/__tests__/handleAction-test.js

```diff
@@ -65,6 +65,22 @@ describe('handleAction()', () => {
             counter: 10
           });
       });
+
+      it('should should support additional arguments', () => {
+        const { increment } = createActions('INCREMENT');
+
+        const reducer = handleAction(
+          increment, (state, { payload }, extraArg) => ({
+            counter: state.counter + payload + extraArg
+          }),
+          { counter: 3 });
```

Comment:
Can we put the closing parenthesis on a newline, and use the same number of arguments in `handleActions-test.js`?